package.json

```diff
@@ -53,9 +53,9 @@
     "eslint-config-standard": "^11.0.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-mocha-no-only": "^1.1.0",
-    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-node": "^10.0.0",
```

Comment:
Can you please undo the changes in `package.json`? We prefer to have PRs that focus on separate issues.